packages/terra-date-picker/src/react-datepicker/index.jsx

```diff
@@ -51,84 +51,297 @@ const WrappedCalendar = onClickOutside(Calendar)
 
 class DatePicker extends React.Component {
   static propTypes = {
+    /**
+     * Prop to change date when a valid date is selected.
+     */
     adjustDateOnChange: PropTypes.bool,
+    /**
+     * Prop to check for same dates.
```

Comment:
Can be "Prop to enable/disable same dates."

Reply:
Resolved in 25cbd8b